<?xml version="1.0"?>
<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
        <ros2_control name="RealRobot" type="system">
            <hardware>
                <plugin>robertito/RobertitoDriver</plugin>
	    </hardware>
	   <xacro:swerve_drive_ros2_control name="front_right"/>
	   <xacro:swerve_drive_ros2_control name="front_left"/>
	   <xacro:swerve_drive_ros2_control name="rear_right"/>
	   <xacro:swerve_drive_ros2_control name="rear_left"/>
        </ros2_control>
</robot>
